components/BottomNavigation/README.md

```diff
@@ -181,3 +181,56 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+## Accessibility {#a11y}
+
+To help ensure your bottom navigation item is accessible to as many users as possible, please
+be sure to review the following recommendations:
+
+### `-accessibilityLabel`
+The label will be the title of the UITabBarItem. Currently you can't set this to a custom value.
+
+### Set `-accessibilityValue` 
+
+Set an appropriate
+[`accessibilityValue`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619583-accessibilityvalue)
+value if your item has a badge value. For example, an item with an inbox icon with a badge value for how many
```

Comment:
The default is the `badgeValue`. I think we could advise clients about when to set it explicitly. What do you think?